src/core/DisplayGroup.h

```diff
@@ -109,6 +131,12 @@ public slots:
      */
     void setBackgroundContent( ContentPtr content );
 
+    /**
+     * Set the the value of the showWindowTitles option.
+     * @param set The setter for showWindowTitles option.
```

Comment:
'set' seems right to me since this function can Enable AND Disable the showWindowTitles option. No ?
PS: I removed the double 'the'